### ISSUE TITLE
3.3.5 Character info issues

### ISSUE THREAD

**Issue by Zergei** [Macumbafeh/Fizzle-modified#1]
Probably its one of my addons causing this issue but still wanted to report it 
![image](https://github.com/user-attachments/assets/d0354c88-8df8-484b-be55-5e75f15589f4)

![image](https://github.com/user-attachments/assets/20367c09-60bb-4a54-98c1-2336c44fe3ef)

i disabled all of my other addons and enable only fizzle but the character info frames looks way too BIG 
![image](https://github.com/user-attachments/assets/5b0478a7-8510-480b-a5f3-760f00d7fbaa)

**Comment by Macumbafeh** [comment]
Yeah the frame is bigger to match with the rest. It was made like this, so if an other addon change the size of it it will change everything too.
